dimik-descending-number/en.md

```diff
@@ -0,0 +1,27 @@
+# Dimik - Descending Number
+
+In this problem,you will not be given any inputs.The only thing which you will have to do is to print all the numbers starting from 1 upto 1000 but in descending order.  
+
+### Solution 
+* Each number has to be seperated using the `space` character.
+* Each line will consist of exactly 5 numbers not more nor less. 
+  
+### C++
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+int main()
+{
+    int cnt = 0;
+    for (int i = 1000; i >= 1; i--)
+    {
+        cout << i << " ";
```

Comment:
Although this passes in LightOJ, which I wonder how, but this is printing trailing spaces after each line. The goal was to only print spaces **between** numbers. Let's please do just that and not print trailing spaces.